sandbox/pages/HomePage.vue

```diff
@@ -81,23 +83,45 @@ const filteredComponents = computed(() => {
       type: 'multi-color',
       name: key,
       component: val,
+      keyword: '',
     })
   }
 
   // flags
   for (const [key, val] of Object.entries(flagIcons)) {
+    interface Country {
+      code: string
+      name: string
+    }
+
+    // Create a map of 2 letter code to country name
+    const countryMap: Map<string, { name: string }> = COUNTRY_CODES.reduce((
+      map: Map<string, { name: string }>,
+      country: Country,
+    ) => map.set(country.code, { name: country.name }), new Map())
+
+    // Grab 2-letter country code from icon name
+    const match = /Flag(.*?)Icon/.exec(key) || ''
+    const countryCode = match[1].toUpperCase()
+
     allComponents.push({
       type: 'flags',
       name: key,
       component: val,
+      keyword: countryMap.has(countryCode) ? countryMap.get(countryCode)?.name : '',
     })
   }
 
   if (!searchQuery.value || searchQuery.value?.toLowerCase() === 'icon') {
     return allComponents
   }
 
-  return allComponents.filter((icon: any) => icon.name.toLowerCase().includes(searchQuery.value.toLowerCase().replace(/icon/gi, '')))
+  const searchTerm = searchQuery.value.toLowerCase().replace(/icon/gi, '')
+
+  return allComponents.filter((icon: any) => {
+    return icon.name.toLowerCase().includes(searchTerm) ||
+      icon?.keyword?.toLowerCase().includes(searchTerm)
```

Comment:
If you make `keywords` an array you will have to remove `.toLowerCase()` from here but I suggest you transform all keywords to lowercase when you're adding them to the `keywords` array instead (line 111)